Formula/naga.rb

```diff
@@ -20,6 +20,8 @@ class Naga < Formula
 
   uses_from_macos "ncurses"
 
+  conflicts_with "naga-cli", because: "naga also ships a naga binary"
+
```

Comment:
Let's add the `conflicts_with` separately so we don't have to rebuild these bottles.